llama_index/llms/litellm_utils.py

```diff
@@ -0,0 +1,260 @@
+import logging
+import os
+import re
+from typing import Any, Callable, Dict, List, Optional, Sequence, Type, Union
+
+import litellm
+from litellm import completion, acompletion
+
+from llama_index.bridge.pydantic import BaseModel
+
+from tenacity import (
+    before_sleep_log,
+    retry,
+    retry_if_exception_type,
+    stop_after_attempt,
+    wait_exponential,
+)
+
+from llama_index.llms.base import ChatMessage
+
+GPT4_MODELS = {
```

Comment:
While I'm brainstorming how to reduce code duplication in the main LLM file, we should just import these details from `openai_utils.py` if they are the same no?